src/GeositeFramework/js/SidebarToggle.js

```diff
@@ -47,20 +47,84 @@
     N.views.TogglePlugin = Backbone.View.extend({
         initialize: function(options) {
             this.singlePlugin = options.viewModel.get('plugins').at(0);
+            this.viewModel = options.viewModel;
+            this.viewModel.set('pluginContentVisible', true);
+            this.listenTo(this.viewModel, 'change', this.adjustToggleIcon);
         },
 
         events: {
             'click': 'togglePluginVisibility'
         },
 
+        adjustToggleIcon() {
+            const newIconClass =
+                this.viewModel.get('pluginContentVisible') ?
+                'fa fa-chevron-right' : 'fa fa-chevron-left';
+
+            this.$el.find('i').attr('class', newIconClass);
+        },
+
         togglePluginVisibility: function() {
-            if (this.singlePlugin.selected) {
+            if (this.viewModel.get('pluginContentVisible')) {
+                this.viewModel.set('pluginContentVisible', false);
                 this.singlePlugin.deselect();
-                this.$el.find('i').attr('class', 'fa fa-chevron-right');
             } else {
+                this.viewModel.set('pluginContentVisible', true);
                 this.singlePlugin.select();
-                this.$el.find('i').attr('class', 'fa fa-chevron-left');
             }
         }
     });
+
+    // Mobile toggle view for single plugin mode
+    N.views.MobileTogglePlugin = Backbone.View.extend({
+        initialize: function(options) {
+            this.viewModel = options.viewModel;
+            this.mobileContentButton = $('#single-plugin-toggle-content-button');
+            this.mobileMapButton = $('#single-plugin-toggle-map-button');
+
+            this.listenTo(this.viewModel, 'change', this.adjustButtonCSS);
+            this.showMobileMap = this.showMobileMap.bind(this);
+            this.showMobileContent = this.showMobileContent.bind(this);
+
+            this.mobileMapButton.on('click', this.showMobileMap);
+            this.mobileContentButton.on('click', this.showMobileContent);
+
+            N.app.toggleMobileMap = this.showMobileMap;
```

Comment:
I think I prefer the internal (`show...`) name of these functions over the public (`toggle...`) name of these functions, since they will always show and not toggle the appropriate content. Is that right?

Reply:
Yeah -- I'll change these throughout to `show...`.